packages/notte-integrations/src/notte_integrations/sessions/hyperbrowser.py

```
@@ -0,0 +1,65 @@
+            cdp_url=session.ws_endpoint,
+        )
+
+    @override
+    def close_session_cdp(self, session_id: str) -> bool:
+        if self.verbose:
+            logger.info(f"Closing CDP session {session_id}")
+
+        try:
+            _ = asyncio.run(self.client.sessions.stop(session_id))
+            return True
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)